zaza/functests/deploy.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import datetime
+import importlib
+import logging
+import os
+import subprocess
+import sys
+import unittest
+import yaml
+
+import juju_wait
+
+BUNDLE_DIR = "./tests/bundles/"
+DEFAULT_TEST_CONFIG = "./tests/tests.yaml"
+
+def deploy_bundle(bundle, model):
+    """Deploy the given bundle file in the specified model
+
+    :param bundle: str Path to bundle file
+    :param model: str Name of model to deploy bundle in
```

Comment:
These can be:

    :param bundle: Path to bundle file
    :type bundle: str
    :param model: Name of model to deploy bundle in
    :type model: str

and if there is a return

    :returns: Something thruthy
    :rtype: bool

If it returned a None or a str, then it would look like

    :returns: Maybe a string
    :rtype: Option[None, str]